### PR TITLE
fix(orc8r): Updated helm chart version for 1.8.0 release

### PR DESCRIPTION
**Pull request by ShubhamTatvamasi**
## Summary

Updates the helm chart, docker-images and  documentation references to version 1.8.x in order to release the helm chart.

### CODE REVIEW

**Review by crasu — COMMENTED** [review]
@ShubhamTatvamasi the documentation needs an update as well https://github.com/magma/magma/blob/v1.8/docs/readmes/orc8r/deploy_install.md still has 1.6 in it.

**Review by crasu — APPROVED** [review]
@ShubhamTatvamasi the documentation needs an update as well https://github.com/magma/magma/blob/v1.8/docs/readmes/orc8r/deploy_install.md still has 1.6 in it.

**Comment by crasu** [comment]
@ShubhamTatvamasi updated the documentation, squashed and changed target branch to v1.8

**Comment by sebathomas** [comment]
Should this be merged only to v1.8 or also to master?

**Comment by crasu** [comment]
> Should this be merged only to v1.8 or also to master?

needs to be merged to master as  well

**Comment by crasu** [comment]
done

**Review by crasu — APPROVED** [review]

**Comment by crasu** [comment]
@sklgromek Could you have a look at this?

**Review by sklgromek — APPROVED** [review]
LGTM

**Review by alexzurbonsen — APPROVED** [review]
LGTM

File: docs/readmes/orc8r/deploy_intro.md [hunk 1]
```diff
@@ -64,6 +64,15 @@ To target a specific release, checkout the Magma repository's relevant release b
 
 Values for recent Orchestrator releases are summarized below
 
+### v1.8.0
+
+Verified with Terraform version `1.0.11`.
+
+- `v1.8` [patch branch](https://github.com/magma/magma/tree/v1.8)
+- `github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-aws?ref=v1.8`
```
Comment (sebathomas) [inline docs/readmes/orc8r/deploy_intro.md:72]: Is the double slash intentional?

Comment (ShubhamTatvamasi) [reply docs/readmes/orc8r/deploy_intro.md:72]: Yes

**Review by Neudrino — APPROVED** [review]

**Pull request merged**